### PR TITLE
PXWEB2-216 Added Spinner

### PR DESCRIPTION
**Pull request by PerIngeVaaje**
Added Spinner component. Also corrected storybook path for Alert

### CODE REVIEW

File: libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  size = 'medium',
+  variant = 'default',
+  label,
+}: SpinnerProps) {
+  let PathStrokeSubtle = '';
+  let PathStrokeAction = '';
+  let Eclipse = null;
+  let Labelsize: 'small' | 'medium' = 'medium';
+
+  switch (variant) {
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx:20]: This switch and the variables on line 15 and 16 seem to be ununsed. Please remove them.

File: libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    case 'inverted':
+      PathStrokeSubtle = 'var(--px-color-border-subtle)';
+      PathStrokeAction = 'var(--px-color-border-on-inverted)';
+      break;
+    default:
+      PathStrokeSubtle = 'var(--px-color-border-subtle)';
+      PathStrokeAction = 'var(--px-color-border-action)';
+      break;
+  }
+  switch (size) {
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx:34]: Theres a lot of code being repeated in the SVGs here. I'd prefer only changing some variables and then having one svg part, but I'll leave the decision here to you :)

**Review by SjurSutterudSagen — CHANGES_REQUESTED** [review]

File: libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx (hunk 2 above)
Comment (PerIngeVaaje) [reply libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx:22]: Changes done.

**Review by SjurSutterudSagen — APPROVED** [review]
LGTM

**Pull request merged**